app/jsx/components/Nav1Comp.jsx

```diff
@@ -1,20 +1,59 @@
 // ##### Navigation - Home Component ##### //
 
 import React from 'react'
+import { Link } from 'react-router'
 import Breakpoints from '../../js/breakpoints.json'
 
+// When running in the browser (and only then), include polyfill(s)
+if (!(typeof document === "undefined")) {
+  require('details-polyfill')
+}
+
 class Nav1Comp extends React.Component {
   state = { isOpen: true } // must init to something in case of server-side rendering
+
   componentWillMount() {
     if (typeof matchMedia != "undefined") {
       this.mq = matchMedia("(min-width:"+Breakpoints.screen2+")")
       this.mq.addListener(this.widthChange)
       this.widthChange()
     }
+    if (this.props.campuses) {
+      this.campusSelector = this.props.campuses.map(function(c, i) {
+          return c['id'] != "" && <Link key={i} to={"/unit/" + c['id']}>{c['name']}</Link>
+        })
+    }
+  }
+
+  campusListGenerator(props) {
+    if (props.campuses) { return (
+      <div className="c-nav1__sub-items">
+        {this.campusSelector}
+      </div>
+      )}
+    // No props? Just return static version
+    return (
```

Comment:
Ah, I see. That's where we'll have to go, but it would be worth talking through global-state API vs. current-page API. So I'd say just stick a comment in the code for this campus list explaining that we're going to remove it just as soon as we figure out a good way :)